api/src/main/resources/metadata/VisitTypes.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <dataset>
```

Comment:
Eliminate the visit type and create two encounter types. 

1. COVID-19 Case Initiation
2. COVID-19 Case Follow up

Reply:
Resolved